torchaudio/functional/functional.py

```diff
@@ -298,13 +298,81 @@ def DB_to_amplitude(
     return ref * torch.pow(torch.pow(10.0, 0.1 * x), power)
 
 
+def _hz_to_mel(freq: Tensor, mel_scale: str = "htk") -> Tensor:
+    r"""Convert Hz to Mels.
+
+    Args:
+        freqs (Tensor): Frequencies in Hz
+        mel_scale (str, optional): Scale to use: ``htk`` or ``slaney``. (Default: ``htk``)
+
+    Returns:
+        mels (Tensor): Input frequencies in Mels
+    """
+
+    if mel_scale not in ['slaney', 'htk']:
+        raise ValueError('mel_scale should be one of "htk" or "slaney".')
+
+    if mel_scale == "htk":
+        return 2595.0 * torch.log10((700.0 + freq) / 700.0)
```

Comment:
keep `math.log10` as before?

Reply:
Wouldn't that break torchscript compatibility?

Reply:
Yes, though I rewrote the function with the math library and it is also torchscriptable now